File: 2015/1/1.go
package main

import (
    "fmt"
    "io/ioutil"
)

func main() {
	file, err := ioutil.ReadFile("1_in.txt")
	if err != nil {
		panic(err)
	}

	input := string(file)

	count := 0
	i := 0
	for i < len(input) {
		if input[i] == '(' {
			count++
		} else if input[i] == ')' {
			count--
		}
		if count < 0 {
			break
		}
		i++
	}
	fmt.Println(i+1)
}

File: 2018/template/1.js
const _ = require('lodash');
const fs = require('fs');

const IS_TEST = 0;

fs.readFile(IS_TEST ? 'test.txt' : 'in.txt', 'utf8', function(err, contents) {
  let ans;

  // let inputs = contents;

  let input = contents.split('\n');
  input.pop();

  input.forEach(line => {

  });  

  console.log('ans: ', ans);
});
